cookbooks/ambari_metrics/templates/default/ams-grafana.ini.erb

```diff
@@ -0,0 +1,258 @@
+
+##################### Grafana Configuration Example #####################
+#
+# Everything has defaults so you only need to uncomment things you want to
+# change
+
+# possible values : production, development
+; app_mode = production
+
+#################################### Paths ####################################
+[paths]
+# Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
+#
+;data = /var/lib/grafana
+data = <%=node['ams']['grafana']['lib_location']%>
+#
+# Directory where grafana can store logs
+#
+;logs = /var/log/grafana
+logs = <%=node['ams']['grafana']['log_location']%>
+
+
+#################################### Server ####################################
+[server]
+# Protocol (http or https)
+;protocol = http
+protocol = http
+# The ip address to bind to, empty will bind to all interfaces
+;http_addr =
+
+# The http port  to use
+;http_port = 3000
+http_port = 3000
+
+# The public facing domain name used to access grafana from a browser
+;domain = localhost
+
+# Redirect to correct domain if host header does not match domain
+# Prevents DNS rebinding attacks
+;enforce_domain = false
+
+# The full public facing url
+;root_url = %(protocol)s://%(domain)s:%(http_port)s/
+
+# Log web requests
+;router_logging = false
+
+# the path relative working path
+;static_root_path = public
+static_root_path = /usr/lib/ambari-metrics-grafana/public
+
+# enable gzip
+;enable_gzip = false
+
+# https certs & key file
+;cert_file =
+;cert_key =
+cert_file = /etc/ambari-metrics-grafana/conf/ams-grafana.crt
+cert_key = /etc/ambari-metrics-grafana/conf/ams-grafana.key
+
+#################################### Database ####################################
+[database]
+# Either "mysql", "postgres" or "sqlite3", it's your choice
+;type = sqlite3
```

Comment:
How do we make this HA to not rely on sqlite3 or what gets stored in this database?

Reply:
The ams grafana is not used and will be replaced with in-house grafana.